vars/pipelineVegaMarketSim.groovy

```diff
@@ -128,9 +128,14 @@ void call() {
                         }
                         steps {
                             sh label: 'Fuzz Test', script: '''
-                                scripts/run-docker-fuzz-test.sh
+                                scripts/run-docker-fuzz-test.sh ${NUM_FUZZ_TESTS}
                             '''
                         }
+                        post {
+                            success {
+                                archiveArtifacts artifacts: '*.jpg, *.html'
```

Comment:
Are the artefacts available in the root folder or subfolder?

Reply:
Should be in the root folder I think, this is the snippet from the script running the fuzz test.

```
docker rm fuzz_test || true
docker run \
    --platform linux/amd64 \
    --name fuzz_test \
    -v "${RESULT_DIR}:/tmp" \
    vega_sim_learning:latest \
        python -m vega_sim.scenario.fuzzed_markets.run_fuzz_test --steps $1

docker cp fuzz_test:/vega_market_sim/fuzz_plots/. .
docker rm fuzz_test
```